/*
 * This file was automatically generated by EvoSuite
 * Mon Jun 06 06:38:19 GMT 2016
 */

package org.collegeboard.challenge.guessnum;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.collegeboard.challenge.guessnum.Main;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = false) 
public class Main_ESTest extends Main_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Main.main((String[]) null);
      // Undeclared exception!
      try { 
        Main.main((String[]) null);
        fail("Expecting exception: Error");
      
      } catch(Error e) {
         //
         // java.io.IOException: Simulated exception in System.in
         //
         assertThrownBy("org.collegeboard.challenge.guessnum.Ui", e);
      }
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Main main0 = new Main();
  }
}
